apps/web/src/components/ThemeSelector/ThemeSelector.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react'
+import { useTranslation } from 'react-i18next'
+import { Moon, Sun, Palette, Trees, Waves, Mountain, Snowflake, TreePalm } from 'lucide-react'
+import { useTheme } from '../../contexts/ThemeContext'
+import { cn } from 'util/index'
+
+const themeIcons = {
+  base: Palette,
+  forest: Trees,
+  ocean: Waves,
+  desert: Mountain,
+  snow: Snowflake,
+  jungle: TreePalm
+}
+
+export default function ThemeSelector ({ className }) {
+  const { t } = useTranslation()
+  const {
+    currentTheme,
+    setCurrentTheme,
+    colorScheme,
+    setColorScheme,
+    availableThemes
+  } = useTheme()
+
+  return (
+    <div className={cn('flex flex-col gap-4', className)}>
+      <div className='flex items-center justify-between'>
+        <label className='text-sm font-medium'>{t('Color Scheme')}</label>
+        <div className='flex items-center gap-2 rounded-lg border p-1'>
+          <button
+            onClick={() => setColorScheme('light')}
```

Comment:
we should have the option here to choose system, so it continues to change based on the system's dark or light mode choice

Reply:
and that should continue to be the default for folks who have not chosen otherwise